docs/cvl/methods.md

```diff
@@ -446,6 +515,9 @@ was instead `function Token.burn() external envfree;` without a summary,
 the `HAVOC_ALL` of the wildcard entry will apply.
 ```
 
+```{note}
+Dispatch list summary for unresolved calls is not included, as above summaries will only be applied for resolved calls.
```

Comment:
This is not clear?

Reply:
This probably detracts more then it adds so removed.